app/validators/Base.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace GanttDashboard\App\Validators;
+
+use Phalcon\Validation;
+use Phalcon\Validation\Message;
+use Phalcon\Validation\Message\Group;
+
+class Base extends Validation
```

Comment:
are you sure that this is required?
why does not phalcon handle array manipulatins by default for errors?
classes that are not mean to be used as new Class, and their role is to be extended are abstract classes

Reply:
I'll use it for projects too. The result of ->validate($array) methos is an object that can't be displayed in view by keys. It's keys are numeric and in order to display the errors under each field a for would have been needed for each field in volt. The same method does not support other parameters in order to return indexed array by field's name. I'll make that class abstract.

Reply:
if it's an object it's means that it can be accessed, and not viewed by keys. it should be doable else I would not comment

Reply:
this is not a negociation it's an requirement!

Reply:
ok.